### DISCUSSION TITLE
Necessary Dependencies to Get Vulkan Working? Crashing on Hello World Tutorial

### DISCUSSION THREAD

**Discussion by chillhammer** [bevyengine/bevy#3938] [Q&A] [answered]
Hello,
I am working through the Hello World tutorial in the Bevy documentation, but am failing on the step to get a window open.
OS: Windows 10
GPU: Nvidia 3080, latest drivers.
The error is this:

`Finished dev [unoptimized + debuginfo] target(s) in 3.63s
     Running `target\debug\my_bevy_game.exe`
2022-02-13T22:38:31.893043Z  INFO bevy_render::renderer: AdapterInfo { name: "NVIDIA GeForce RTX 3080", vendor: 4318, device: 8710, device_type: DiscreteGpu, backend: Vulkan }
hello Elaina Proctor!
hello Renzo Hume!
hello Zayna Nieves!
hello world!
2022-02-13T22:38:34.301306Z ERROR wgpu_hal::vulkan::instance: VALIDATION [VUID-VkFramebufferCreateInfo-flags-03189 (0x0)]
        vkCreateFramebuffer(): VkFramebufferCreateInfo flags includes VK_FRAMEBUFFER_CREATE_IMAGELESS_BIT_KHR, but the imagelessFramebuffer feature is not enabled. The Vulkan spec states: If the imageless framebuffer feature is not enabled, flags must not include VK_FRAMEBUFFER_CREATE_IMAGELESS_BIT_KHR (https://www.khronos.org/registry/vulkan/specs/1.1-extensions/html/vkspec.html#VUID-VkFramebufferCreateInfo-flags-03189)
2022-02-13T22:38:34.301957Z ERROR wgpu_hal::vulkan::instance:   objects: (type: UNKNOWN, hndl: 0x0, name: ?)
2022-02-13T22:38:34.326609Z ERROR wgpu_hal::vulkan::instance: VALIDATION [VUID-VkRenderPassBeginInfo-framebuffer-03209 (0x0)]
        VkRenderPassBeginInfo: Image view #0 created from an image with flags set as 0x400, but image info #0 used to create the framebuffer had flags set as 0x0 The Vulkan spec states: If framebuffer was created with a VkFramebufferCreateInfo::flags value that included VK_FRAMEBUFFER_CREATE_IMAGELESS_BIT_KHR, each element of the pAttachments member of an instance of VkRenderPassAttachmentBeginInfoKHR included in the pNext chain must be a VkImageView of an image created with a value of VkImageCreateInfo::flags equal to the flags member of the corresponding element of VkFramebufferAttachmentsCreateInfoKHR::pAttachments used to create framebuffer (https://www.khronos.org/registry/vulkan/specs/1.1-extensions/html/vkspec.html#VUID-VkRenderPassBeginInfo-framebuffer-03209)
2022-02-13T22:38:34.329355Z ERROR wgpu_hal::vulkan::instance:   objects: (type: RENDER_PASS, hndl: 0x364d940000000031, name: ?)
error: process didn't exit successfully: `target\debug\my_bevy_game.exe` (exit code: 0xc000041d)`

I have not developed with Vulkan before so I am guessing there is a step that I am missing with that?

**Comment by cart** [comment, marked as answer]
Hmm theres a closed issue on the wgpu repo with a similar error message: https://github.com/gfx-rs/wgpu/issues/2176

**Reply by cart** [reply]
Updating the Vulkan SDK fixed it for them

**Reply by chillhammer** [reply]
Thanks! That did it
